Microsoft.Research/Contracts/System.Core.Contracts/System.Linq.Expressions.SwitchCase.cs

```diff
@@ -44,8 +45,11 @@ sealed public partial class SwitchCase
     #region Methods and constructors
     private SwitchCase() {}
 
+    [Pure]
     public SwitchCase Update (IEnumerable<Expression> testValues, Expression body)
     {
+      Contract.Requires(body != null);
```

Comment:
In this case, null or empty `testValues` are invalid (http://referencesource.microsoft.com/#System.Core/Microsoft/Scripting/Ast/SwitchCase.cs,102):

`Contract.Requires(testValues != null);`
`Contract.Requires(testValues.Any());`

Not sure what approach is better: with one or two statements...